libuavcan/include/libuavcan/build_config.hpp

```diff
@@ -66,6 +66,17 @@
 #    define LIBUAVCAN_EXPORT
 #endif
 
+/**
+ * C++ feature test macros are newer then c++11 so we translate them into a "LIBUAVCAN"
+ * macro to allow us to manually enable exceptions if the compiler doesn't define the __cpp
+ * macro.
+ */
+#ifdef __cpp_exceptions
+#    ifndef LIBUAVCAN_ENABLE_EXCEPTIONS
+#        define LIBUAVCAN_ENABLE_EXCEPTIONS
```

Comment:
Okay. I'll fix

Reply:
I fixed usages of this macro but for the macros in this header ifdef is correct since we are only interested in the "not defined" case and do want to override any "false" cases.

Reply:
Yes, I see that, perfect.

Reply:
There is one last thing though. `#define LIBUAVCAN_ENABLE_EXCEPTIONS` is insufficient because with a value test like `#if LIBUAVCAN_ENABLE_EXCEPTIONS` it results in a malformed program. GCC would say, for example:

```
error: #if with no expression
 #if LIBUAVCAN_ENABLE_EXCEPTIONS
                                ^
```

Every `#define` should specify a non-zero integer for enabled options, zero for disabled options.